test/factory/dom_factory_test.dart

```diff
@@ -27,8 +24,8 @@ main() {
     });
 
     test('has a type corresponding to the DOM tagName', () {
-      expect((react.div as ReactDomComponentFactoryProxy).type, 'div');
-      expect((react.span as ReactDomComponentFactoryProxy).type, 'span');
+      expect((react.div).type, 'div');
```

Comment:
Just cleaning up a lint!

Reply:
```suggestion
      expect(react.div.type, 'div');
```
#nit unnecessary parens